Felienne/hedy
# [FEATURE] Adds language filtering on explore page

**Description**
We already support level and adventure filtering on the explore page but in this PR add the option to filter on language. Only returning the programs that are stored with a user with a profile language set to the selected language.

**Fixes**
This PR fixes #2813.

**How to test**
Make sure you are logged in, navigate to the `/explore` page and verify that language filtering is working.
